tests/foreman/cli/test_repository_set.py

```diff
@@ -17,3 +22,128 @@ def test_available_repository_1(self):
 
         """
         self.fail('This stubbed test should be fleshed out')
+
+    def test_repositoryset_available_repositories(self):
+        """@Test: List available repositories for repository-set
+
+        @Feature: Repository-set
+
+        @Assert: List of available repositories is displayed, with
+        valid amount of enabled repositories
+
+        """
+        rhel_product_name = 'Red Hat Enterprise Linux Server'
+        rhel_repo_set = (
+            'Red Hat Enterprise Virtualization Agents '
+            'for RHEL 6 Server (RPMs)'
+        )
+
+        # Clone manifest and upload it
+        org = make_org()
+        manifest = manifests.clone()
+        upload_file(manifest, remote_file=manifest)
+        result = Subscription.upload({
+            u'file': manifest,
+            u'organization-id': org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+
+        # No repos should be enabled by default
+        result = RepositorySet.available_repositories({
+            u'name': rhel_repo_set,
+            u'organization-id': org['id'],
+            u'product': rhel_product_name,
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(
+            sum(int(repo['enabled'] == u'true') for repo in result.stdout),
+            0
+        )
+
+        # Enable repo from Repository Set
+        result = RepositorySet.enable({
+            u'name': rhel_repo_set,
+            u'organization-id': org['id'],
+            u'product': rhel_product_name,
+            u'releasever': '6Server',
+            u'basearch': 'x86_64',
+        })
+        self.assertEqual(result.return_code, 0)
+
+        # Only 1 repo should be enabled
+        result = RepositorySet.available_repositories({
+            u'name': rhel_repo_set,
+            u'organization': org['name'],
+            u'product': rhel_product_name,
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(
+            sum(int(repo['enabled'] == u'true') for repo in result.stdout),
+            1
+        )
```

Comment:
Again, can we use `all` to directly check the values of these booleans instead of type-casting the booleans to an integer and summing the integers together?